src/TicketBai/Mews.Fiscalizations.TicketBai/Model/SendInvoiceResponse.cs

```diff
@@ -0,0 +1,49 @@
+﻿using FuncSharp;
+using System;
+using System.Collections.Generic;
+
+namespace Mews.Fiscalizations.TicketBai.Model
+{
+    public sealed class SendInvoiceResponse
+    {
+        public SendInvoiceResponse(
+            string xmlRequestContent,
+            string qrCodeUri,
+            string tbaiIdentifier,
+            DateTime received,
+            string state,
+            string description,
+            string stateExplanation,
+            string csv,
+            IEnumerable<SendInvoiceValidationResult> validationResults = null)
+        {
+            XmlRequestContent = xmlRequestContent;
+            QrCodeUri = qrCodeUri;
+            TBAIIdentifier = tbaiIdentifier;
+            Received = received;
+            State = state;
+            Description = description;
+            StateExplanation = stateExplanation;
+            CSV = csv.ToNonEmptyOption();
+            ValidationResults = validationResults.ToOption();
+        }
+
+        public string XmlRequestContent { get; }
+
+        public string QrCodeUri { get; }
+
+        public string TBAIIdentifier { get; }
+
+        public DateTime Received { get; }
+
+        public string State { get; } // TODO: enum?
+
+        public string Description { get; }
+
+        public string StateExplanation { get; }
```

Comment:
Maybe also an enum?

Reply:
This will contain a text explaining the state.

Reply:
Sure, but there's like a predefined list of answers. So it theoretically could be like an Enum. Hey, we'll see if there's something later on.